### PR TITLE
Update tests for BrowserStack

### PR DESCRIPTION
**Pull request by ijjk**
Update tests to setup webdriver stuff in `jest-environment` and re-use one browser session instead of spawning one for each webdriver call to prevent creating too many BrowserStack sessions.

### CODE REVIEW

File: test/integration/production/test/security.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 import { renderViaHTTP, getBrowserBodyText, waitFor } from 'next-test-utils'
-import webdriver from 'next-webdriver'
 import { recursiveReadDir } from 'next/dist/lib/recursive-readdir'
 import { homedir } from 'os'
 
 // Does the same evaluation checking for INJECTED for 15 seconds, triggering every 500ms
 async function checkInjected (browser) {
   const start = Date.now()
-  while (Date.now() - start < 15000) {
+  while (Date.now() - start < 10000) {
```
Comment (Timer) [inline test/integration/production/test/security.js]: Why was this changed? 🤔

Comment (ijjk) [reply test/integration/production/test/security.js]: Was just testing

**Review by Timer — APPROVED** [review]
